src/components/browse/dataset/dataset.tsx

```diff
@@ -18,7 +18,7 @@ const Dataset = (props: dataSetListProps) => {
                             <Accordion.Button className='bg-light align-items-start fs-7'>
                                 <p className="my-0">
                                     <span className="fw-bold">Dataset ID:&nbsp;</span>
-                                    <a href="#action" title={hit.id}>{hit.id} <sup className="text-dark"><BoxArrowUpRight /></sup></a>
+                                    {hit.content.accession}
```

Comment:
Please remove Line no4 since we are getting rid of the BoxArrowUpRight component